### DISCUSSION TITLE
Pausing Pro-Projects

### DISCUSSION THREAD

**Discussion by TheOtherBrian1** [supabase/supabase#27399] [Troubleshooting]
_This is a copy of a troubleshooting article on Supabase's docs site. It may be missing some details from the original. View the [original article](https://supabase.com/docs/guides/troubleshooting/pausing-pro-projects-vNL-2a)._

Pro-Projects at the moment cannot be paused. However, [You are allowed to have two free organizations](https://supabase.com/docs/guides/platform/billing-on-supabase#free-plan) that can support one active project each and an unlimited amount of paused ones.

If a project is under 500MB, you can [transfer it to be under a free organization](https://supabase.com/docs/guides/platform/project-transfer). Afterwards, you can initiate a pause.

Alternatively, you can download a [daily backup](https://supabase.com/dashboard/project/_/database/backups/scheduled) of only your database for archiving. You can also manually download a .SQL file of your database and storage buckets by following this [guide](https://supabase.com/docs/guides/platform/migrating-within-supabase/backup-restore).

You can also download your storage buckets with the [Supabase CLI:](https://supabase.com/docs/guides/local-development/cli/getting-started?queryGroups=platform\&platform=npx)

```sql
npx supabase login

# link to your project
npx supabase link

npx supabase init

# will download files to current folder
npx supabase storage cp -r ss://bucket . --experimental
```

**Comment by JediPixels** [comment]
I hope the feature to pause projects on paid plans is implemented in the future.

**Reply by chrissy0** [reply]
me too

**Comment by oscar-o-oneill** [comment]
I don't understand why we can't pause projects on a paid plan? It doesn't make any sense. We should be able to pause projects on a paid plan. What if you're just testing out something and want to pause work on it for a bit? I understand there being a cost to having a paused project, but it should be less than running a full compute and having to pay for it each month...

**Comment by MisaelCalvillo** [comment]
Is super annoying to have to do all of this just to stop paying that instance if I want to stop it. It seems that you are adding friction intentionally for people to have a billing each month on instances they forget to pause. You should implement an easy way to stop instances and stop paying for them if you want. Just allow us to take a project to Nano in Pro Plan. 

You have such a complex billing that you need a page to understand it. Should be simpler, you pay for what you want to use. https://supabase.com/docs/guides/platform/billing-on-supabase#organization-based-billing

**Comment by adamjdavidson** [comment]
This is the most confounding, maddening business decision I've seen in years. 

I have to assume that a lot of supabase users are working on many versions of a project or projects in various states of production. 

That we can not pause any versions is ... insane. 

I am forced to delete multiple projects so that I don't pay a lot of money every month. 

I can't think of another service that gives LESS freedom to paid subscribers than free ones. I can't think of another service that is so blind to the daily use of its product. 

This is a crucial moment in the development of database services. I know I'm not the only one now eagerly looking for alternatives to Supabase because of this one issue.

**Reply by RyzorBent** [reply]
This is very strange, ... i also just want to pause a project which is currently not being used, ... but by the sound of things i have to delete and loose all the configs; This is ridiculous.

@SupabaseTeam ... please stop this madness.

**Comment by roofleDev** [comment]
Adding to the list of frustrated users wishing to pause a stale project.

**Comment by felixmacaspac** [comment]
Still no updates with this one? damn. 😅

**Comment by artemcluster** [comment]
any news?

**Reply by SHULMIT** [reply]
ממש סיוט הדבר הזה

**Comment by m3h0w** [comment]
I just want to add my frustration into this thread. It's ridiculous that a free user is treated better than a subscribed one. I was super locked-in on using Supabase for all my projects but I'm angry at this and will be trying out Convex instead.

**Comment by saltcod** [comment]
Hey all — appreciate the feedback.
This has complexity on both the infrastructure and billing sides, so it isn’t something we’re planning to support at the moment. As others mentioned, projects under 500 MB can be transferred into one of your free organizations and paused there.

**Comment by JediPixels** [comment]
Hey @saltcod,

Thank you for the update. We also need to clarify that a project paused for too long cannot be recovered after a certain point.

**Reply by saltcod** [reply]
You can always download an export from the dashboard, but yes, after 90 days a project can't be restored directly. 
https://supabase.com/docs/guides/platform/migrating-within-supabase/dashboard-restore

**Comment by crycodebaby** [comment]
I get punished by giving them $25 a month 
what the actual fuck guys
